gcp_audit_rules/gcp_gcs_public.py

```diff
@@ -33,5 +33,5 @@ def dedup(event):
 
 
 def title(event):
-    'GCS bucket [{}] made public'.format(event['resource'].get(
+    return 'GCS bucket [{}] made public'.format(event['resource'].get(
```

Comment:
This is exactly the kind of typo/bug we were hoping to catch with the `panther_analysis_tool` change!